fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1741,6 +1741,9 @@ public class Config extends ConfigBase {
     @ConfField
     public static String starmgr_s3_sk = "";
 
+    @ConfField
+    public static long shard_group_clean_interval = 3600000L;
```

Comment:
this is not an `interval`, it is an expiration or threshold, suggest name: `shard_group_clean_threshold_ms`